pkg/util/docker/docker_util.go

```diff
@@ -145,6 +145,23 @@ func (d *dockerUtil) dockerImages(includeIntermediate bool) ([]types.ImageSummar
 	return images, nil
 }
 
+// countVolumes returns the number of attached and dangling volumes.
+func (d *dockerUtil) countVolumes() (int, int, error) {
+	attachedFilter, _ := buildDockerFilter("dangling", "false")
+	danglingFilter, _ := buildDockerFilter("dangling", "true")
+
+	attachedVolumes, err := d.cli.VolumeList(context.Background(), attachedFilter)
+	if err != nil {
+		return 0, 0, fmt.Errorf("unable to list docker volumes: %s", err)
```

Comment:
"unable to list attached docker volumes"? same bellow for dangling might be useful if filters can trigger error